packages/Uploader/src/components/DropZone/DropZone.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import tokens from "@paprika/tokens";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import UploadIcon from "@paprika/icon/lib/Upload";
+import { UploaderContext } from "../../Uploader";
+import * as div from "./DropZone.styles";
+
+export default function DropZone() {
+  const I18n = useI18n();
+  const uc = React.useContext(UploaderContext);
+  const isDraggingOver = uc.isDraggingOver;
+  const uploadIconColor = isDraggingOver ? tokens.color.purpleDarken10 : tokens.color.blackLighten50;
+
+  /* eslint-disable jsx-a11y/anchor-is-valid */
+  const body = isDraggingOver ? (
+    I18n.t("uploader.drop_files")
+  ) : (
+    <React.Fragment>
+      {I18n.t("uploader.drop_files_here_or")}&nbsp;
+      <a
```

Comment:
This should be a `<Button kind="link">`. I know you've flagged `a11y` as a `TODO`, but this is a quick refactor, and hard for me to let a practice like this slip through.

Reply:
I originally had it as a button, but it gets ugly on narrow window size - it won't split the text mid-button.

Reply:
> it won't split the text mid-button

Good...?

Seems to me like breaking "Choose files..." into two lines would look worse than breaking the line before it anyway?

Reply:
Oh, I see that it's longer, "Choose from your computer... " 🤔

Reply:
Resize using anchor (pretty):
![nice-resize-anchor](https://user-images.githubusercontent.com/23224777/83194949-51027e80-a0ee-11ea-8cb4-22b30fc194ac.gif)

Resize using `<Button type="link">` (ugly):
![bad-resize-button](https://user-images.githubusercontent.com/23224777/83194956-5364d880-a0ee-11ea-96fa-a8ec4843f0d0.gif)